### PR TITLE
Turtl

### PR DESCRIPTION
**Pull request by bdmorin**
##### Instructions
- Look for and complete the section relevant to your submission. Delete the others, including these `Instructions`.
- `{{cask_file}}` represents the cask file you’re submitting/editing (if applicable).
- If there’s a checkbox you can’t complete for any reason, that’s OK. Just explain in detail why you weren’t able to do so.
#### Editing an existing cask
- [ ] Commit message includes cask’s name (and new version, if applicable).
- [ ] `brew cask audit --download {{cask_file}}` is error-free.
- [ ] `brew cask style --fix {{cask_file}}` left no offenses.
#### Adding a new cask
- [x] Checked there aren’t open [pull requests](https://github.com/caskroom/homebrew-cask/pulls) for the same cask.
- [x] Checked there aren’t closed [issues](https://github.com/caskroom/homebrew-cask/issues) where that cask was already refused.
- [x] When naming the cask, followed the [token reference](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md).
- [x] Commit message includes cask’s name.
- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` left no offenses.
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.
### Changes to the core
- [ ] Followed [hacking.md](https://github.com/caskroom/homebrew-cask/blob/master/doc/development/hacking.md).

### CODE REVIEW

**Comment by vitorgalvao** [comment]
You’re removing an unrelated cask.

File: Casks/turtl.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+cask 'turtl' do
+  version :latest
```
Comment (vitorgalvao) [inline Casks/turtl.rb]: `version '0.6.4'`

File: Casks/turtl.rb [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+  sha256 :no_check
```
Comment (vitorgalvao) [inline Casks/turtl.rb]: `sha256` also needs to be set. You can find what it is by running `shasum -a 256` on the downloaded file.

File: Casks/turtl.rb [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+
+  url 'https://turtl.it/releases/desktop/turtl-osx-0.6.4.dmg'
```
Comment (vitorgalvao) [inline Casks/turtl.rb]: `url "https://turtl.it/releases/desktop/turtl-osx-#{version}.dmg"`

**Comment by bdmorin** [comment]
Apologies, i've updated with version and sha256

**Comment by vitorgalvao** [comment]
You’re still removing `putio-adder`.

**Comment by bdmorin** [comment]
arrghh.. how did putio-adder get in there. sorry

**Comment by bdmorin** [comment]
I'm trying to remove that change, it was from a long time ago, and I have no idea what I did. Apologies for the mess. I don't mean to be touching putio-adder, just adding turtl.

**Comment by vitorgalvao** [comment]
No worries, I think I can fix it.

**Comment by vitorgalvao** [comment]
Thank you for the contribution. It needed some fixes, so they were made in 7f750e45f937ca47026dd466957014162b8035e6. Your contribution is still included (and still credited to you), with the appropriate modifications. Please feel free to ask about any of the changes.

**Comment by bdmorin** [comment]
Thank you for your help!

**Pull request closed without merging**